### ISSUE TITLE
Simple 8085

### ISSUE THREAD

**Issue by RizThomas** [TomNisbet/Simple8085#3]
Hi Tom…. Well after rebuild on a protoboard, I got it working ( somehow ).
Somehow it is very unstable… I thought if I I set my FTDI at 9600 baud, hit reset then it displays something.
It does but garbage characters until I get the right baud rate.
So maybe I should type few characters upon reset… no luck … hit and miss
Maybe I am not doing the right steps?

**Comment by TomNisbet** [comment]
The monitor is expecting a space character to allow it to automatically detect the baud rate.  Set for 9600, hit reset, and then press the spacebar and it should come up.  I'll work that into the documentation somewhere.

**Comment by RizThomas** [comment]
Yes!! Thanks for the reply. It was waiting for the SPACE char. I was typing H… (@9600 ) then puts out garbage char then change to 14400…. Seems stable but….
Now it WORKS.

side note: i added a resistor network on pins 6,8,9,10,39 pin 7 to SID but it did not change any behaviour of the baud rate.
THANK YOU … happy to have my 2nd 8085

**Comment by 9600** [comment]
Hello, you called?

**Comment by RizThomas** [comment]
No .. but left a reply here….no issues now… will be creating a case for S8085
 Cheers

**Comment by RizThomas** [comment]
On the 8085, pins 6, (7), 8, 9, and 39 are connected to GND.  Instead I placed 1K ohm resistors in between the pins to GND. Except for pin 7 is now connected with pin 5 SID.I saw this on Gitchworks and Altaids schematic. Without knowing the about “Spacebar” char needed as input, I thought there was s instability in the circuit…. Of course I know nothing…. I left the RN since it doesn’t affect at allSent from my iPhoneOn Oct 5, 2023, at 9:02 AM, Tom Nisbet ***@***.***> wrote:﻿
@RizThomas - where did you add a resistor network? Not sure how or where extra resistors would impact the baud rate.
